pkg/client/monitoring/v1alpha1/types.go

```diff
@@ -73,6 +73,8 @@ type PrometheusSpec struct {
 	RoutePrefix string `json:"routePrefix,omitempty"`
 	// Storage spec to specify how storage shall be used.
 	Storage *StorageSpec `json:"storage,omitempty"`
+	// A selector to select which ConfigMaps to mount for loading rule files from.
+	RuleSelector *metav1.LabelSelector `json:"ruleSelector,omitempty"`
```

Comment:
Definitely! I came up with:

```yaml
role: rulefiles
prometheus: main
```

If that sounds plausible then I'll add it to the documentation.